schema.graphql

```diff
@@ -126,12 +128,10 @@ type Transfer @entity {
   from: Account!
   to: Account!
   value: BigInt!
```

Comment:
"value" should potentially be "amount" to match deposits/withdrawals. Though, I could go either way on this, since the transaction actually uses "value" as a key